exec/java-exec/src/main/resources/rest/storage/update.ftl

```diff
@@ -53,8 +53,17 @@
     });
     function doUpdate() {
       $("#updateForm").ajaxForm(function(data) {
-        $("#message").removeClass("hidden").text(data.result).alert();
-        setTimeout(function() { location.reload(); }, 800);
+        var messageEl = $("#message");
+        messageEl.addClass("hidden");
+        // Wait a fraction of a second before showing the message again. This
+        // makes it clear if a second attempt gives the same error as
+        // the first that a "new" message came back from the server
+        setTimeout(function() {
+          messageEl.removeClass("hidden").text("Please retry: " + data.result).alert();
```

Comment:
Originally I didn't have the "Please retry:" I had added that after finding your patch. Previously, it would just show either the error or the success message and only in the case of the success message reload. Will push a fix